index.js

```diff
@@ -50,17 +50,19 @@ module.exports = opts => {
 	const pkg = ret.pkg || {};
 	const pkgPath = ret.path || path.resolve(opts.cwd || '', 'package.json');
 	const pkgCwd = path.dirname(pkgPath);
-	const s = pkg.scripts = pkg.scripts ? pkg.scripts : {};
+	const s = pkg.scripts || {};
 
 	if (s.test && s.test !== DEFAULT_TEST_SCRIPT) {
-		// don't add if it's already there
+		// Don't add if it's already there
 		if (!/^xo( |$)/.test(s.test)) {
 			s.test = `xo && ${s.test}`;
 		}
 	} else {
 		s.test = 'xo';
 	}
 
+	pkg.scripts = s;
```

Comment:
Personally, I'm not sure if this would improve readability.

@kevva What do you think about refactoring the whole `if` out? For example, something like:

```js
const buildTestScript = oldTest => {
	if (oldTest && oldTest !== DEFAULT_TEST_SCRIPT) {
		// Don't add if it's already there
		if (!/^xo( |$)/.test(oldTest)) {
			return `xo && ${oldTest}`;
		}
		return oldTest;
 	}
	return 'xo';
};

module.exports = opts => {
	…
	pkg.scripts = pkg.scripts || {};
	pkg.scripts.test = buildTestScript(pkg.scripts.test);
	…
}
```